operators/pkg/controller/elasticsearch/validation/validations.go

```diff
@@ -149,3 +151,36 @@ func tlsCannotBeDisabled(ctx Context) validation.Result {
 	}
 	return validation.OK
 }
+
+// pvcModification ensures no PVCs are changed, as volume claim templates are immutable in stateful sets
+func pvcModification(ctx Context) validation.Result {
+	if ctx.Current == nil {
+		return validation.OK
+	}
+	for _, node := range ctx.Proposed.Elasticsearch.Spec.Nodes {
+		currNode := getNode(node.Name, ctx.Current.Elasticsearch)
+		if currNode == nil {
+			// this is a new sset, so there it is okay
+			return validation.OK
```

Comment:
What will happen if first sset is new and the next one is already existing but with modified volume claim templates?

Reply:
A bug :)

I'll fix